portfolio/src/main/webapp/script.js

```diff
@@ -16,14 +16,58 @@ function getComments() {
   fetch('/data').then(response => response.json()).then((commentsArray) => {
     let container = document.getElementById('comments-container');
     container.innerText = '';
+    console.log(commentsArray);
     for (const comment of commentsArray) {
-        container.appendChild(createListElement(comment));
+        container.append(createCommentView(comment));
     }
   });
 }
 
+function initMap() {
+  const prefPlace = { lat: 55.17477877, lng: 61.29954625 };
```

Comment:
It's an exercise, there's no requirement to post personal data. Non-personal data is fine, or even better.

Reply:
Of course it is not a place where I live, but it is it near enough anyway I don't think it can be called personal

Reply:
Anyway made it less personal